### ISSUE TITLE
Help page should respect dark theme

### ISSUE THREAD

**Issue by siraben** [tridactyl/tridactyl#1121]
The Tridactyl help page should respect the setting of `theme` when set to `dark`.

**Comment by bovine3dom** [comment]
This was added a few months ago, I believe.

**Comment by glacambre** [comment]
The help pages do not react to theme changes yet.

**Comment by rprimus** [comment]
Sun Oct 11 18:02:18 BST 2020

Hi all,

Any update on this?

**Comment by bovine3dom** [comment]
No, but it should be fairly straightforward to add.

Adapt https://github.com/tridactyl/tridactyl/blob/5e8f94ff06bd5f143732f2dabd32b08dabf1e65a/src/static/css/newtab.css#L1 and put it somewhere in this file https://github.com/tridactyl/tridactyl/blob/5e8f94ff06bd5f143732f2dabd32b08dabf1e65a/src/static/typedoc/assets/css/main.css#L9

: )

**Comment by rprimus** [comment]
Mon Oct 12 10:32:24 BST 2020

Have made a start - base on above suggestions.

https://github.com/rprimus/tridactyl/pull/new/help-page-follows-theme

As mentioned on `matrix`:
1. 'static/clippy/1-tutor.html' works correctly (ie follows colors), however 'static/docs/modules/_src_excmds_.html#tridactyl-help-page' doesn't

<img width="1680" alt="tridactyl-greenmat-tutor" src="https://user-images.githubusercontent.com/278414/95731018-84454480-0c6e-11eb-940a-0f858dc39370.png">

Only 'outer' background is changed.  'navigation'-fg,  'help-inner'-bg seem to be hardcoded.

<img width="1680" alt="tridactyl-greenmat-help" src="https://user-images.githubusercontent.com/278414/95731028-87d8cb80-0c6e-11eb-9283-010f800f9ee1.png">

Will trackdown later on.

2. on `macOS`:

```sh
: ; git ci -m 'Make help page follow colour scheme'
> /usr/local/src/tridactyl/hooks/pre-commit exists, running

mktemp: illegal option -- -
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix 
```

**Comment by bovine3dom** [comment]
Doesn't the tutor get the colour from the theme already?

**Comment by rprimus** [comment]
Mon Oct 12 11:59:21 BST 2020

Yes, it does - just using it to compare.

Q:  What is a 'module'?  I see they are created in `background.ts`.

**Comment by bovine3dom** [comment]
https://github.com/tridactyl/tridactyl/blob/5e8f94ff06bd5f143732f2dabd32b08dabf1e65a/src/background.ts#L31 here? That's just a bunch of functions essentially. I don't think it's relevant to this : )

Could you open a PR so we can discuss there? I don't want to ping people who have subscribed to this issue too much.

**Comment by rprimus** [comment]
Mon Oct 12 12:38:14 BST 2020

Success?

<img width="1680" alt="tridactyl-help-dark" src="https://user-images.githubusercontent.com/278414/95742379-75b35900-0c7f-11eb-8c8c-5f30c8823906.png">
<img width="1680" alt="tridactyl-help-halloween" src="https://user-images.githubusercontent.com/278414/95742386-79df7680-0c7f-11eb-8656-41ec6a538d9f.png">
<img width="1680" alt="tridactyl-help-greenmat" src="https://user-images.githubusercontent.com/278414/95742388-7a780d00-0c7f-11eb-8202-c6985607d17e.png">
<img width="1680" alt="tridactyl-help-quake" src="https://user-images.githubusercontent.com/278414/95742391-7b10a380-0c7f-11eb-919e-5cbe048fd708.png">


Will update my fork.  Currently, I've added the CSS below it's corresponding section in the original file - so that it overides.

Please let me know how you'd like the final changes to be applied - ie as I've done it, or actually modify the current config - changing what's currently there (vs overide).

**Comment by bovine3dom** [comment]
Fixed in #2895
